fe/fe-core/src/main/java/com/starrocks/external/hive/HiveMetaClient.java

```diff
@@ -57,16 +64,33 @@ private void init() {
         this.baseHmsEventId = currentNotificationEventId.getEventId();
     }
 
+    public static HiveMetaClient createHiveMetaClient(Map<String, String> properties) {
+        HiveConf conf = new HiveConf();
+        properties.forEach(conf::set);
+        if (!properties.containsKey(HIVE_METASTORE_URIS)) {
+            // set value for compatible with the rollback
+            properties.put(HIVE_METASTORE_URIS, DUMMY_THRIFT_URI);
```

Comment:
why use this ？ The properties must have "hive.metastore.uris".

Reply:
glue doesn't have this property